hikari/internal/enums.py

```diff
@@ -395,7 +402,6 @@ def __new__(
             "_member_names_": (member_names := []),
             # Required to be immutable by enum API itself.
             "__members__": types.MappingProxyType(namespace.names_to_values),
-            **namespace,
```

Comment:
Moving this to the bottom will achieve the same effect as `new_namespace.update`

Reply:
While relying on implementation detail on the order in which those are handled and being less explicit that it's meant to be handled after the fields are brought in from Flag